modules/understanding-upgrade-channels.adoc

```diff
@@ -7,17 +7,156 @@
+.  {product-version}.4
+
+The service only recommends upgrades that have been tested and have no serious issues.
+If you are on {product-version}.1 and {product-title} is allowing you to select
+{product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise,
+the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no
+longer suggests upgrading to a known vulnerable version.
+
+Update stability depends on your channel. The presence of an update recommendation in
+the candidate-{product-version} channel does not imply that the update is supported.
+It means that no serious issues have been found with the update yet, but there may
+not be significant traffic through the update to suggest stability. The presence of
+an update recommendation in the fast-{product-version} or stable-{product-version}
+channels is a declaration that the update is fully supported while it is in the
+channel. Like releases in the stable-{product-version} channel, update
+recommendations in the stable-{product-version} channel will be promoted from
+fast-{product-version} with a delay that can range from several hours to a day.
+While releases will never be removed from a channel, update recommendations which
+exhibit serious issues will be removed from all channels. Updates initiated after
+the update recommendation has been removed may not be supported.
+
+Red Hat will eventually provide supported update paths from any supported (fast-{product-version}
+or stable-{product-version}) release to the latest release in {product-version}.z,
+although there may be delays while safe paths away from troubled releases are
+constructed and verified.
+
+= Fast and Stable Channel Usage and Strategies
+
+The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+general availability releases as soon as they are available or allowing Red Hat to
+control the rollout of those updates. If issues are detected during rollout, upgrades
+to that version may be blocked in both the fast-{product-version} and stable-{product-version}
+channels, and a new version may be introduced that will be the new preferred upgrade
+target.
+
+Customers can improve this process by configuring pre-production systems on the
+fast-{product-version} channel, production systems on the stable-{product-version} channel,
+and participating in Red Hat’s connected customer program. This program allows Red
+Hat to observe the impact of updates on your specific hardware and software
+configurations. Future releases may improve or alter the pace at which updates move
+from the fast-{product-version} to the stable-{product-version} channel.
+
+If issues are discovered with a release between patch levels, Red Hat may withdraw
+that suggested upgrade path for affected release versions. A newer release would
+become available in the appropriate channels and be suggested as an upgrade path.
```

Comment:
This paragraph overlaps with the earlier "If issues are detected during rollout..." line and the even earlier "While releases will never be removed from a channel, update recommendations which exhibit serious issues will be removed from all channels." paragraph.  I think we can drop this one.

Reply:
Fixed in c3ce8cd.